.github/workflows/_helm_lint_publish.yml

```diff
@@ -37,27 +37,23 @@ jobs:
           - v1.23.0
           - v1.24.0
     steps:
-      - name: Checkout
+      - name: Run the helm conform
         uses: LedgerHQ/actions/helm/conform@add-reusable-workflow-helm
         with:
           kubernetes-version: ${{ matrix.k8s }}
-          charts-path: ${{inputs.charts-path}}
+          charts-path: ${{ inputs.charts-path }}
         
   publish-chartmuseum-dev:
     name: publish-chartmuseum-dev
-    environment: chartmuseum-dev
+    environment: dev
     if: inputs.version != ''
     runs-on: [self-hosted, shared]
     needs: [lint-chart, kubeconform-chart]
     steps:
-      - name: Checkout
```

Comment:
why ?

Reply:
the checkout is done already in the helm cm-push action